test/Bruno/.env.sample

```
@@ -8,4 +8,4 @@ TOKEN_URL=
 # Use the country code in URL encoded format
 EMAIL_ADDRESS_TEST=test@test.no
 PHONE_NUMBER_TEST=99999999
-COUNTRY_CODE_TEST=%2B47
\ No newline at end of file
+COUNTRY_CODE_TEST=+47
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)